From: Miod Vallat <miod@tarides.com>
Date: Wed, 19 Jun 2024 13:13:03 +0000
Subject: Rework fixed register assignments on m68k.

The use of %a5 conflicts with its internal usage as a GOT base by Glibc,
so shift assignments one register down.

Origin: https://github.com/ocaml/ocaml/pull/13252
---
 runtime/interp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/interp.c b/runtime/interp.c
index 66f0123..05c61e3 100644
--- a/runtime/interp.c
+++ b/runtime/interp.c
@@ -209,7 +209,7 @@ Caml_inline void check_trap_barrier_for_effect
 #define ACCU_REG asm("%r16")
 #endif
 #ifdef __mc68000__
-#define PC_REG asm("a5")
+#define PC_REG asm("a3")
 #define SP_REG asm("a4")
 #define ACCU_REG asm("d7")
 #endif
